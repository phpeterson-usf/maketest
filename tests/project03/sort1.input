sort 87 53 63 23 204 8 42 0
