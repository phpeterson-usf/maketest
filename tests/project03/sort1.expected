C: 204 87 63 53 42 23 8 0
Assembly: 204 87 63 53 42 23 8 0
